README.md

```diff
@@ -1,30 +1,31 @@
 ![Hireable](https://img.shields.io/badge/Hireable-yes-success)
 
 ![](https://img.shields.io/badge/-Microverse%20projects-blueviolet)
-# Positioning and Floating Elements project: New York Times article page
+# Microverse project: Using Bootstrap - Replica of the news site Newsweek.com
 
-> This is a New York Times article page clone using HTML and CSS3.
+> This is a Newsweek homepage page clone using HTML and CSS3.
 
-## Screenshots of original website
+## Screenshots of our clone website
 
-![screenshot](./app-screenshot-original-full.png)
+![screenshot](./.github/app-screenshot.png)
 
-## Screenshots of our clone website
+## Link to the original website
 
-![screenshot](./app-screenshot-clone-full.png)
+[Original version](https://www.newsweek.com/)
 
 
-The project is built using Html and Css with an extensive use of the grid, flex and float css properties.
+The project is built using Html and CSS with an extensive use of the Bootstrap CSS framework, flex and grid positioning.
 
 ## Built With
 
 - HTML
 - CSS3
+- Bootstrap
 - Font awesome icons
 
 ## Live Demo
 
-[Live Demo Link](https://raw.githack.com/cristianCeamatu/microverse-new-york-times-article-page/adding-article-page/index.html)
+[Live Demo Link](https://rawcdn.githack.com/cristianCeamatu/microverse-news-week-clone/5f371f1d3ec3732f46862c5fd46c91b6e66aaa40/index.html)
```

Comment:
- [ ] **[AFTER MERGING]** Please update your link, it is showing your page without the corrections you did. As a suggestion, so you don't have to be manually updating this link during your code reviews, use the second link (development link) in githack, that one will automatically load your last version in the same branch :)